pages/article/[id].js

```diff
@@ -39,6 +39,38 @@ const useStyles = makeStyles(theme => ({
       alignItems: 'baseline',
     },
   },
+  title: {
+    position: 'relative',
+    left: -8,
+    fontWeight: 'bold',
+    backgroundColor: theme.palette.primary[500],
+    color: theme.palette.secondary.main,
+    padding: '10px 32px',
+    [theme.breakpoints.down('md')]: {
+      padding: '10px 16px',
+      fontSize: 12,
+    },
+    '&:after': {
+      position: 'absolute',
+      top: 0,
+      right: 0,
+      borderRight: `20px solid ${theme.palette.common.white}`,
+      borderBottom: `20px solid ${theme.palette.primary[500]}`,
+      borderTop: `20px solid ${theme.palette.primary[500]}`,
```

Comment:
Unfortunately the tails are not aligned in narrower screen:
![image](https://user-images.githubusercontent.com/108608/85220902-e3422d80-b3e1-11ea-8c1f-719c6cd6f15a.png)
(Actually the one in the menu is not aligned either.)

Since it's difficult calculating these triangles, I would suggest using svg instead.
It's shape is so simple that we can write the svg [by hand](https://www.oxxostudio.tw/articles/201406/svg-04-path-1.html):
```
<svg viewBox="0 0 1 2">
  <path d="M0 0 H1 L0 1 L1 2 H0 Z" />
</svg>
```
And we get a ribbon tail that we can resize and recolor using CSS.